### DISCUSSION TITLE
seems better if you use Content-Type header in node/express

### DISCUSSION THREAD

**Discussion by grahampcharles** [dolanmiu/docx#1053] [Ideas / Requests]
This is super-minor but I thought I'd drop it here in case anyone else searches up the issue in the future. The node/express demos on RunKit ([like this one](https://runkit.com/dolanmiu/docx-demo10)) indicate that browsers will correctly detect `Content-Type` from the file extension. Here's the relevant code from that example:

```
    res.setHeader('Content-Disposition', 'attachment; filename=My Document.docx');
    res.send(Buffer.from(b64string, 'base64'));
```

For whatever reason, Chrome 91 -- at least on my machine -- kind of chokes on that. It drops the document correctly, but the browser returns a home page window with the viewport scaled to like 300%. Interestingly, some keyboard shortcuts (like Zoom and Show/Hide Toolbar) are also disabled. 

Anyway, I'm sure it's all Chrome's fault, but it seems safer to just send the explicit header.

```
res.setHeader(
            "Content-Type",
            "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
        ).setHeader(
            "Content-Disposition",
            "attachment; filename=My Document.docx"
        );
```
Chrome deals with that correctly.

**Comment by dolanmiu** [comment]
Thanks for sharing!
